core/store/data_input.hpp

```diff
@@ -142,6 +148,8 @@ struct index_input : public data_input {
   //////////////////////////////////////////////////////////////////////////////
   virtual int64_t checksum(size_t offset) const = 0;
 
+  virtual void CountMemory(MemoryStats stats) const {}
```

Comment:
We also have to implement it for fs/memory

Reply:
Ok